datafusion/sqllogictest/test_files/array_query.slt

```diff
@@ -108,11 +108,15 @@ SELECT * FROM data WHERE column2 is not distinct from null;
 # Aggregates
 ###########
 
-query error Internal error: Min/Max accumulator not implemented for type List
+query ?
 SELECT min(column1) FROM data;
```

Comment:
Can we also add tests for empty input on min/max as well to test boundary conditions? Something like
```sql
SELECT MIN(column1) FROM data WHERE false
```

Which should return a null value?